### ISSUE TITLE
Page Number code?

### ISSUE THREAD

**Issue by rjbprime** [jsnee/vscode-dmbinder#15]
Trying to change the colour of the pagenumber auto in my brew using CSS. I have written ```color: black;``` in the relevant place, and it's defaulting to a gold colour and a different size from what I chose, which seems to be the same as the default Homebrewery PHB style. Would it be possible to change this to allow the change?

Attached are the PDF and renamed md files, to see if they help.

[A4.pdf](https://github.com/jsnee/vscode-dmbinder/files/4455934/A4.pdf)

[A4.md.txt](https://github.com/jsnee/vscode-dmbinder/files/4455938/A4.md.txt)

**Comment by jsnee** [comment]
No problem, I'll take a look at it and get back to you! :)

**Comment by rjbprime** [comment]
May have worked out the issue. It seems to be a ID10T error, as I didn't realise that the code I was using **had not** defined the colour and size, and it was reading the default homebrewery code. I worked it out by copying my code to homebrewery itself, then used Chrome's Code Inspector to check the code. Sorry for the trouble over this issue, I'll close it now.

Can the label be changed to "User (ID10T) Error"? Then I can have a laugh at my own expense.

**Comment by rjbprime** [comment]
Got a new issue, may be related, may be my code. Trying to disable the footer image on the last page of a brew, using

```
.phb#p33:after{
	background: none;
	background-image: none;
	content:"";
}
```

isn't working, yet the first page one, which is 

```
.phb#p1:after{
	background: none;
	background-image: none;
	content:"";
}
```

**is** working. Copying the code into homebrewery itself does work, for both p33 and p1, with the footer image on all other pages showing.

Attached are my PDF and MD files, to see if they help.

[AiGr.md.txt](https://github.com/jsnee/vscode-dmbinder/files/4469952/AiGr.md.txt)

[AiGr_compressed.pdf](https://github.com/jsnee/vscode-dmbinder/files/4469957/AiGr_compressed.pdf)

**Comment by jsnee** [comment]
Fixed in latest version (0.5.8). As it turns out, the first page is numbered "p1" but then the rest of the pages are numbered in reverse order (in your case, "p33" was the table of contents). Version 0.5.8 has all the pages in correct ascending order.

That'll teach me to borrow code from the internet... Or not, haha
